packages/gui/src/electron/utils/openExternal.ts

```diff
@@ -0,0 +1,11 @@
+import { shell } from 'electron';
+
+import isValidURL from './isValidURL';
+
+export default function openExternal(url: string) {
+  if (!isValidURL(url)) {
```

Comment:
I think `openExternal` is one of the most dangerous API in the GUI.
I think currently we don't allow arbitral url and only  uses to access fixed list of domains like `github.com`, `feedback.chia.net` etc. Those are all `https`.
In my brief investigation, we don't open page with `http://` nor `ipfs://` at all.

I don't say that we should create white list of allowed domains but I think we can disallow `http` and `ipfs`  here, which `isValidURL()` allows currently.

Reply:
I removed http, we have support fot https and ipfs in cloud wallet as well